pkg/apis/componentconfig/types.go

```diff
@@ -684,9 +684,11 @@ type KubeControllerManagerConfiguration struct {
 	ConcurrentSATokenSyncs int32
 	// lookupCacheSizeForRC is the size of lookup cache for replication controllers.
 	// Larger number = more responsive replica management, but more MEM load.
+	// DEPRECATED: This is no longer used.
```

Comment:
Can you add these same comments to pkg/apis/componentconfig/v1alpha1/types.go and also a timeline for when it can be safely removed?